lib/state.dart

```diff
@@ -113,71 +114,99 @@ class AppState with ChangeNotifier {
     return success;
   }
 
-  Future<bool> sendReport(Map<String, dynamic> symptoms) async {
-    var success = false;
+  Future<bool> _submitSymptoms(
```

Comment:
@joshgummersall Just realized I had a bit of a duplicate effort here: #35

Reply:
Yeah, I'd say get that other PR merged up then you can take this one over.